src/main/java/at/hannibal2/skyhanni/features/slayer/enderman/EndermanSlayerHideParticles.kt

```diff
@@ -14,7 +14,7 @@ import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
 
 class EndermanSlayerHideParticles {
 
-    private var endermanLocations = listOf<LorenzVec>()
```

Comment:
are you sure this works? sequences can only be consumed once

Reply:
No i am not and it didnt work at all